command.go

```diff
@@ -264,8 +265,15 @@ func stripFlags(args []string) []string {
 				inQuote = true
 			case strings.Contains(y, "=\""):
 				inQuote = true
+			case strings.HasPrefix(y, "--") && !strings.Contains(y, "="):
+				inFlag = true
+			case strings.HasPrefix(y, "-") && len(y) == 2:
```

Comment:
What about if I mix the flags together? For instance, boolean flag `-a` and boolean flag `-b`, I can write `-ab` for it...